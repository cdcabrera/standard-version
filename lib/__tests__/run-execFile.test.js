const runExecFile = require('../run-execFile');

describe('run-execFile', () => {
  it('should attempt to execFile commands', async () => {
    const dryRunResults = await runExecFile({ dryRun: true });
    const echoCmdResults = await runExecFile({}, 'echo', ['lorem ipsum']);

    expect({
      dryRunResults,
      echoCmdResults
    }).toMatchSnapshot('command results');
  });
});
