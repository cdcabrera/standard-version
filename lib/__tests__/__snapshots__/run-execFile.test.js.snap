// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`run-execFile should attempt to execFile commands: command results 1`] = `
Object {
  "dryRunResults": undefined,
  "echoCmdResults": "lorem ipsum
",
}
`;
